dev/AppNotifications/AppNotifications.idl

```diff
@@ -111,6 +111,29 @@ namespace Microsoft.Windows.AppNotifications
 
         // Gets or sets whether a Notification's pop-up UI is displayed on the user's screen.
         Boolean SuppressDisplay;
+
+        // Gets or sets the Notification Device Data
+        AppNotificationDevicesData DevicesData;
```

Comment:
In another world, this should be about _generic_ app notification enhancements... like "here's an HWND you should splat into the toast" or "here's an adaptive card to render with a callback URI" or "here's a COM object you should cocreate to get the stuff to show."

And then "calling" is a specific subtype of that notification enhancement...

Reply:
@jonwis Please review this comment where we have explained how Contoso Calling would interact with WinAppSDK APIs and the XML update. 
https://github.com/microsoft/WindowsAppSDK/pull/4783#issuecomment-2422815364

Please let us know if any update required on the XML, AppNotificationBuilder, Naming Conventions, etc.